sandbox/src/mocks/availableActionsForUserList/Empty.json

```diff
@@ -11,7 +11,7 @@
 	"link": [
 		{
 			"relation": "self",
-			"url": "_baseUrl_/STU3/Task?focus=ReferralRequest/000000070004/_history/6&intent=proposal&status=ready"
+			"url": "_baseUrl_/STU3/Task?focus=ReferralRequest/000000070000/_history/6&intent=proposal&status=ready"
```

Comment:
Because I reordered the table in the description to facilitate adding more scenarios to this endpoint, the empty example will now be 0000 0000 7000 (previously ending in 4)